docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
@mapuri Yes. types.VolumeCreateRequest isn't compatible with VolumeRequest. And, the  build failed because of the structure compatibility. We can stick with the old structure.

@erikh Seems like a lot changed in Response and Request https://github.com/docker/go-plugins-helpers/blob/master/volume/api.go As discussed I'm just moving the structures.